tests/test_gatk4spark.yml

```diff
@@ -49,7 +49,7 @@
     - path: results/preprocessing/mapped/
       should_exist: false
 - name: Run default pipeline with gatk4spark and skipping all QC steps
-  command: nextflow run main.nf -profile test_cache,use_gatk_spark --skip_tools fastqc,markduplicates_report,mosdepth,multiqc,samtools --outdir results
+  command: nextflow run main.nf -profile test_cache,use_gatk_spark,spark --skip_tools fastqc,markduplicates_report,mosdepth,multiqc,samtools --outdir results
```

Comment:
I missed a bunch of discussion around this. What is tl;dr that we have a double spark profile?

Reply:
So 1 for the test, and one for the docker.userEmulation that restore proper usage

Reply:
ah makes sense, so users should set `spark` when runnin it now?

Reply:
makes sense :)